tests/local_mode/test_doc.py

```diff
@@ -108,3 +108,19 @@ def test_update_custom_attr_doc():
 
     # now check the updated attribute
     assert hasattr(doc._, "my_custom_tag") and doc._.my_custom_tag == "new_tag"
+
+
+def test_exclude_tokens_on_attr_values_doc():
+    """Test that the get_vector method ignores tokens based on the excluded_tokens dict"""
+    doc = nlp("Joey doesnt ever share food ")
+    doc_excluding_tokens = nlp("Joey doesnt share food")
+
+    # add custom_attr to the last token, the word ever
+    token = doc[2]
+    token.set_attribute(name="attribute1_name", value="value1")
+
+    # initialize excluded excluded_tokens dict
+    excluded_tokens = {"attribute1_name": ["value1", "value2"], "attribute2_name": ["v1", "v2"]}
+
+    assert all(doc.get_vector(excluded_tokens) == doc_excluding_tokens.get_vector())
```

Comment:
can you please explain these two asserts in comments

Reply:
For the first one, I'm going with:
"checks if get_vector returns the same vector for doc and the doc with the word to exclude already missing, all() is needed because equals for numpy arrays returns an array of booleans."

For the second one:
"checks if get_vector without excluded_tokens returns a different vector for doc and doc with the word to exclude already missing."

They might be a bit too long but I cannot find a better way to explain them.:sweat_smile: 

Btw tell me if more tests are needed.

Reply:
Thank you for the explanations :) I actually meant explaining as code comments

Reply:
Yep, these comments now are in the code :thumbsup:

Reply:
Great